CODEOWNERS

```diff
@@ -24,3 +24,62 @@
 /pkg/monitor/payload @cilium/api
 /pkg/policy/api/ @cilium/api
 /pkg/proxy/accesslog @cilium/api
+/pkg/redirectpolicy @cilium/sig-lb
```

Comment:
None of these codeowners changes should be backported.

Reply:
For older branches, we only set codeowners for APIs, security and otherwise tophat. Where did these get bumped?

Reply:
Commit dropped.

Reply:
dropped https://github.com/cilium/cilium/pull/20619 ✅ these files don't exist in v1.11 branch.

Reply:
I think dropping all of #20619 is a bit much- yes the codeowners change can be dropped but https://github.com/cilium/cilium/blob/v1.11/test/k8sT/l7_demos.go does exist (note `test/k8sT` vs `test/k8s`), and so do the manifests.

Reply:
:+1: https://github.com/cilium/cilium/pull/23313#issuecomment-1403713116